bin/ycsb

```diff
@@ -43,40 +46,46 @@ DATABASES = {
     "mongodb-async": "com.yahoo.ycsb.db.AsyncMongoDbClient",
     "nosqldb"      : "com.yahoo.ycsb.db.NoSqlDbClient",
     "orientdb"     : "com.yahoo.ycsb.db.OrientDBClient",
-    "redis"        : "com.yahoo.ycsb.db.RedisClient", 
+    "redis"        : "com.yahoo.ycsb.db.RedisClient",
     "voldemort"    : "com.yahoo.ycsb.db.VoldemortClient",
     "couchbase"    : "com.yahoo.ycsb.db.CouchbaseClient"
 }
 
 OPTIONS = {
-    "-P file"      : "Specify workload file",
-    "-p key=value" : "Override workload property",
-    "-s"           : "Print status to stderr",
-    "-target n"    : "Target ops/sec (default: unthrottled)",
-    "-threads n"   : "Number of client threads (default: 1)",
+    "-P file"         : "Specify workload file",
+    "-p key=value"    : "Override workload property",
+    "-s"              : "Print status to stderr",
+    "-target n"       : "Target ops/sec (default: unthrottled)",
+    "-threads n"      : "Number of client threads (default: 1)",
+    "-cp path"        : "Additional Java classpath entries",
+    "--jvm-args args" : "Additional arguments to the JVM",
```

Comment:
above we use single dash for long-form arguments, and here we're using gnu style '--'. I don't have a strong preference one way or the other, but we should be consistent.